Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml

```
@@ -612,14 +612,104 @@ tasks:
           root: Qualys.Endpoint
           accessor: ID
       val3:
-        simple: N/A
+        simple: n/a
     separatecontext: false
     continueonerrortype: ""
     view: |-
       {
         "position": {
-          "x": 320,
-          "y": 1210
+          "x": 690,
+          "y": 1360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: c737255d-4571-477a-8887-b29ea923efc1
+    type: regular
+    task:
+      id: c737255d-4571-477a-8887-b29ea923efc1
+      version: -1
+      name: Set ASM - Enrichment Status
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format. For example:
+        `!GridFieldSetup keys=ip,src,timestamp val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" val3="TIMESTAMP" gridfiled="gridfield"`
+      scriptName: GridFieldSetup
+      type: regular
+      iscommand: false
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      gridfield:
+        simple: asmenrichmentstatus
+      keys:
+        simple: source,record_exists,timestamp
+      val1:
+        simple: Qualys
+      val2:
+        simple: "true"
+      val3:
+        simple: TIMESTAMP
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: ea5e1f1e-027e-47ab-8765-0eec317445d0
+    type: regular
+    task:
+      id: ea5e1f1e-027e-47ab-8765-0eec317445d0
+      version: -1
+      name: Set ASM - Enrichment Status
```

```suggestion
      name: Set ASM enrichment status to false
```
Consistency with other playbooks